PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/sqlserver/SqlServerConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.sqlserver;
+
+import com.google.common.base.Preconditions;
+import com.microsoft.azure.management.resources.ResourceGroup;
+import com.microsoft.azure.management.resources.Subscription;
+import com.microsoft.azure.toolkit.lib.appservice.DraftResourceGroup;
+import com.microsoft.azure.toolkit.lib.common.model.Region;
+import com.microsoft.azuretools.core.mvp.model.AzureMvpModel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.DateFormatUtils;
+
+import java.util.Date;
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+public class SqlServerConfig {
+
+    private Subscription subscription;
+    private ResourceGroup resourceGroup;
+
+    private String serverName;
+    private Region region;
+
+    private String adminUsername;
+    private char[] password;
+    private char[] confirmPassword;
+
+    private boolean allowAccessFromAzureServices;
+    private boolean allowAccessFromLocalMachine;
+
+    public static SqlServerConfig getDefaultConfig(Subscription subscription) {
+        final String defaultNameSuffix = DateFormatUtils.format(new Date(), "yyyyMMddHHmmss");
+        final SqlServerConfig config = new SqlServerConfig();
+        config.setSubscription(subscription);
+        DraftResourceGroup resourceGroup = DraftResourceGroup.builder().subscription(subscription).name("rs-" + defaultNameSuffix).build();
```

Comment:
DraftResourceGroup -> ResourceGroup

Reply:
please merge resourcegroup/region/subs lib as soon as you can. I will refactor the code according by it.